### PR TITLE
Embeddings: improve admin page headers

### PR DESCRIPTION
**Pull request by jtibshirani**
This adds the icon and path to the headers for the embeddings jobs and policies pages. This matches what we do for Analytics.

Quick PR for fun, not high priority for release :)

## Test plan

Visual inspection! The styling is a slightly different than for Analytics (icon isn't blue), but I liked how this turned out.

Screenshot 2023-06-09 at 9 39 30 AM

### CODE REVIEW

File: client/web/src/enterprise/cody/configuration/pages/CodyConfigurationPage.tsx [hunk 1]
```diff
@@ -85,6 +86,7 @@ export const CodyConfigurationPage: FC<CodyConfigurationPageProps> = ({
             <PageHeader
                 headingElement="h2"
                 path={[
+                    { icon: mdiRobotOutline, text: 'Cody' },
```
Comment (toolmantim) [inline client/web/src/enterprise/cody/configuration/pages/CodyConfigurationPage.tsx:89]: Is there a Cody icon we can use here?

**Review by toolmantim — COMMENTED** [review]
😍

**Comment by jtibshirani** [comment]
I gave that a try using the two Cody icons I could find ...
Screenshot 2023-06-09 at 2 42 45 PM

File: client/web/src/enterprise/cody/configuration/pages/CodyConfigurationPage.tsx (hunk 1 above)
Comment (jtibshirani) [reply client/web/src/enterprise/cody/configuration/pages/CodyConfigurationPage.tsx]: I gave that a try using the two Cody icons I could find ...
Screenshot 2023-06-09 at 2 42 45 PM

**Review by camdencheek — APPROVED** [review]
Nice!

**Pull request merged**